### PR TITLE
Add Kick icon

### LINKED ISSUE #8908: Kick (streaming)
### Brand Name

Kick

### Website

https://kick.com/

### Popularity Metric

Similar web rank is `665`
https://www.similarweb.com/website/kick.com/#overview

### Official Resources for Icon and Color

https://www.dropbox.com/sh/zzjhpjpzdiegwz9/AAC7FV6oLeHtb9eRruLQ-_p2a?dl=0

### Additional Comments

There's a white and green version of the logo

### Contributing

No

### PR DESCRIPTION
**Pull request by Gautam25Raj**
kick

**Issue:** closes #8908

**Similarweb rank:**
https://www.similarweb.com/website/kick.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from Official Icons in Brand website [Kick](https://www.dropbox.com/sh/zzjhpjpzdiegwz9/AAC7FV6oLeHtb9eRruLQ-_p2a?dl=0)
Color: `#53FC19` Taken from official logo.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6282,6 +6282,11 @@
                 "type": "GPL-3.0-or-later"
             }
         },
+        {
+            "title": "Kick",
+            "hex": "53FC19",
+            "source": "https://www.dropbox.com/sh/zzjhpjpzdiegwz9/AAC7FV6oLeHtb9eRruLQ-_p2a?dl=0"
```
Comment (adamrusted) [inline _data/simple-icons.json:6288]:
```suggestion
            "source": "https://kick.com"
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm spotting [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/afd24c0a-384a-4659-b23d-ab1bae942f01) between the source SVG in green, and your PR in black. Could you have another go at this please @Gautam25Raj 🙏

File: _data/simple-icons.json (hunk 1 above)
Comment (Gautam25Raj) [reply _data/simple-icons.json:6288]: The dropbox link is officially mentioned by [kick](https://kick.com) in their brand section in footer. Do I still need to change them?

Comment (adamrusted) [reply _data/simple-icons.json:6288]: Yep - we're much better placed to link to their website (which has the obvious 'brand' link) as opposed to a random Dropbox URL.

**Comment by Gautam25Raj** [comment]
Sure, I u will check and update it.

**Comment by PeterShaggyNoble** [comment]
Any update on this, @Gautam25Raj?

**Review by adamrusted — APPROVED** [review]

**Pull request merged**